test-web-library-build/src/foo.test.ts

```diff
@@ -0,0 +1,9 @@
+/// <reference types="mocha" />
```

Comment:
No. That is a different issue. For that we will need to upgrade to 1.0.0 of karma and activate another flag.